docs/resources/cluster.md

```diff
@@ -187,19 +187,6 @@ resource "mongodbatlas_cluster" "cluster-test" {
   }
 }
 ```
-### Example AWS Shared Tier (M2/M5) cluster
```

Comment:
on Advanced Cluster page Tenant example https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/advanced_cluster#example-tenant-cluster can we also update example from "M5" to "M0"? all good from my end after you make this update

Reply:
Done!